docs/source/claim_state_machine.dot

```diff
@@ -1,16 +1,15 @@
 digraph claim_state_machine {
-    node [] claimer_winning, challenger_winning, invalidated, invalidated_l1_resolved, ignored, withdrawn;
+    node [] started, claimer_winning, challenger_winning, invalidated_l1_resolved, ignored, withdrawn;
+
+    started -> claimer_winning [label = start_challenge];
+    started -> challenger_winning [label = start_challenge];
```

Comment:
I think this is not correct. `start_challenge()` always transitions from started to claimer_winning

Reply:
Totally.

Reply:
Fixed.